client/src/core/utils.ts

```diff
@@ -118,3 +118,12 @@ export function baseAdjust(url: string): string {
     if (url.startsWith("/")) url = url.slice(1);
     return process.env.BASE_URL + url;
 }
+
+export async function getErrorReason(response: Response): Promise<string> {
+    const responseText: string = await response.text();
+    const parts: string[] = responseText.split(":");
+    if (parts.length > 1) {
+        return parts[1].trim();
+    }
```

Comment:
The response text that I'm seeing come back is `401: Username and/or Password do not match` I assume we just want the part after the ':'. the trim is there to remove the space. I've added a comment explaining the flow get.

The if is there to make sure an an error message still get's returned if the function get something that isn't in the main format it expects.

Reply:
sorry I was apparently just blind, I completely missed the return in the if part.

Reply:
that said, you probably do want to join everything from 1 to the end in case the error message itself would contain a `:` parts might actually be longer than 2!

Reply:
Sounds good. Thinking about that case, it seems better to find the first ':', and drop everything before that, so I updated it to that instead of a split and join. same end result, just seems more clear.